lib/resources/r4/condition.yaml

```diff
@@ -0,0 +1,119 @@
+  binding:
+    description: The clinical status of the condition.
+    terminology:
+      - display: Condition Clinical Status Codes
+        system: http://terminology.hl7.org/CodeSystem/condition-clinical
+        info_link: https://hl7.org/fhir/r4/valueset-condition-clinical.html
+
+- name: verificationStatus
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: unconfirmed | provisional | differential | confirmed | refuted | entered-in-error
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.verificationStatus
+  binding:
+    description: The verification status to support or decline the clinical status of the condition or diagnosis.
+    terminology:
+      - display: Condition Verification Status Codes
+        system: http://terminology.hl7.org/CodeSystem/condition-ver-status
+        info_link: https://hl7.org/fhir/r4/valueset-condition-ver-status.html
+
+- name: category
+  required: 'No'
+  cardinality: 0..*
+  type: CodeableConcept
+  description: problem-list-item | encounter-diagnosis
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.category
+  example: |
+    {
+      "category": {
+        "coding": [
+          {
+            "system": "https://terminology.hl7.org/CodeSystem/condition-category",
+            "code": "problem-list-item",
+            "display": "Problem List Item"
+          }
+        ],
+        "text": "Problem List Item"
+      }
+    }
+  binding:
+    description: A category assigned to the condition.
+    note: Category code `problem-list-item` is supported for search functionality.
+    terminology:
+      - display: Condition Category Codes
+        system: http://terminology.hl7.org/CodeSystem/condition-category
+        info_link: https://hl7.org/fhir/r4/valueset-condition-category.html
+
+- name: severity
+  required: 'No'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Severe | Moderate | Mild
+  url: https://hl7.org/fhir/r4/condition-definitions.html#Condition.severity
+  example: |
+    {
+      "severity": [
+        {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "255604002",
+              "display": "Mild"
+            }
+          ]
+        }
+      ]
+    }
+  example2: |
+    {
+      "type": [
+        {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/12022",
+              "code": "3293",
+              "display": "Mild"
+            }
+          ]
+        }
+      ]
+    }
+  binding:
+    description: A subjective assessment of the severity of the condition as evaluated by the clinician.
+    terminology:
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+        info_link: https://hl7.org/fhir/r4/snomedct.html
+      - display: Millennium Condition Severity
+        system: http://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/12022
```

Comment:
proprietary codes have the https:// link, right?

Reply:
yeah they have https

Reply:
Fixed. [84da6ef]